### ISSUE TITLE
[Q] Parse TODOs from Notes?

### ISSUE THREAD

**Issue by olegbc1** [pbek/QOwnNotes#2230]
#### General question

Is there a way to pickup all "TODO:" items from my notes and place them into the actual todos along with the relevant metadata like tags and note titles?

**Comment by pbek** [comment]
No, sorry. That would be hard to achieve... Basically for the same reasons like in https://github.com/pbek/QOwnNotes/issues/2228#issuecomment-913782743.

**Comment by pbek** [comment]
I myself basically use checkbox lists in markdown files as my Todo lists. And only if I really need a reminder I select the text in the note and open to Todo dialog to create a todo item with a reminder (and maybe insert a link to the todo item as reference into my note).

**Comment by olegbc1** [comment]
That would work for me as well, as I don't actually need the reminders and don't need the tasks to show up in the calendar. I just wanted a central place where I can see all of the TODO items from across my notes so I can quickly click on it and go see the context in the notes. 

Appreciate your work on QON and responding to all the queries here so quickly!

**Comment by pbek** [comment]
I have a "general" TODO note with markdown checkbox lists and for bigger projects a TODO section in the note itself.

**Comment by pbek** [comment]
> Appreciate your work on QON and responding to all the queries here so quickly!

Thank you 😁
